meson.build

```diff
@@ -86,7 +86,6 @@ features = {
 	'libseat': false,
 	'x11-backend': false,
 	'xwayland': false,
-	'xcb-errors': false,
```

Comment:
Hmm, while not a huge problem, this does give slightly reduced feature visibility.

Reply:
Yeah, I agree. `features` is used for public-facing features since it's exposed through pkgconfig and project variables. Maybe we can have something like `private_features` (TODO: find a better name) for dependencies like xcb-errors?

Reply:
Done